### DISCUSSION TITLE
Lara-light Figma Theme colors not matching Code

### DISCUSSION THREAD

**Discussion by mikeritchh** [primefaces/community#1867] [Figma UI Kit]
Why does the Lara Light theme not match the Lara-light-blue code?
See images:
![image](https://github.com/primefaces/community/assets/169296789/1b425b97-3528-4cef-bdf8-ad547d56f5d4)
![image](https://github.com/primefaces/community/assets/169296789/149c37f4-1e57-4ea9-bd20-adefa70c0a3c)

**Comment by onursenture** [comment]
Hello, Yes, the Surface colors and surface-ground have changed but have not been updated yet. This does not affect any component used in this system because we currently use the shade colors defined in Tokens Studio and include them in variables.scss instead of the surface colors in our current color system. We have corrected the Surface colors and surface-ground to be released in the next version. Thank you for reporting this.

<img width="1212" alt="Screenshot 2024-05-10 at 09 54 20" src="https://github.com/primefaces/community/assets/244957/6413b3f8-08c0-427d-9314-cd69ee3bdb32">
